### PR TITLE
fix(unique page views): removed unnecessary classes

### PR DESCRIPTION
**Pull request by GusBaamonde**
fix-unique-pages-views

### CODE REVIEW

File: src/components/Reports/Reports.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ import { Loading } from '../Loading/Loading';
 import { addDays, getStartOfDate } from '../../utils';
 
 // This value means the today date
-const periodSelectedDaysDefault = 0;
+const periodSelectedDaysDefault = 7;
```
Comment (cbernat) [inline src/components/Reports/Reports.js]: With this the default is one week, is it intended?

**Comment by cbernat** [comment]
subarashii 素晴しい ❤️

File: src/components/Reports/Reports.js (hunk 1 above)
Comment (cbernat) [reply src/components/Reports/Reports.js]: @GusBaamonde told me it's intended.

**Comment by cbernat** [comment]
For testing here in prod https://cdn.fromdoppler.com/doppler-webapp/build1837/#/reports

File: src/components/Reports/Reports.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/components/Reports/Reports.js]: yes, maybe need change the comment

**Pull request merged**